install/kubernetes/helm/istio/values.yaml

```diff
@@ -440,6 +440,22 @@ global:
   #
   meshNetworks: {}
 
+  # Specifies the global locality load balancing settings. One of failover or distribute can be set.
```

Comment:
do you mean user can set either failover or distribute but not both?  can you please clarify as this comment is going to our doc.

Reply:
Yes, either or, not both. What would be more clear? I am happy to change it but "one of" makes sense to me so not sure what makes sense to others

Reply:
how about -  
"Locality-weighted load balancing allows administrators to control the distribution of traffic to endpoints based on the localities of where the traffic originates and where it will terminate. Please set either failover or distribute configuration but not both."

Reply:
Sounds good. I was trying to writing the minimal docs needed so we don't duplicate all of the docs from istio.io, but I think that is useful